packages/core/getUserAvailability.ts

```diff
@@ -99,15 +99,41 @@ const getUser = (where: Prisma.UserWhereInput) =>
 
 type User = Awaited<ReturnType<typeof getUser>>;
 
-export const getCurrentSeats = (eventTypeId: number, dateFrom: Dayjs, dateTo: Dayjs) =>
-  prisma.booking.findMany({
+const getUserBookings = async (userId: number) => {
```

Comment:
Why do we need these changes in getUserAvailability? 

I saw an issue if I booked a seat, it kept showing that all seats area available for this slot. Might be because of these changes here

Reply:
During getAvailableSlot function call getUserAvailablity is also called which it self calls the getCurrentSeat function. The getCurrentSeat function is responsible for fetching the reserved seats. getUserAvailability function is resposible for fetching all the free time and free seats also reserved seat which it got from getCurrentSeat. So the change in getUserAvailability in short is responsible for fetching the correct bookingUid for logged in or registering booker which is used for rescheduling and editing user info.

Reply:
Also the issue you got i am not able to reproduce it. I can upload loom vid if you would like. 😊